addons/account_voucher/migrations/9.0.1.0/post-migration.py

```diff
@@ -83,15 +83,54 @@ def create_payments_from_vouchers(env):
         AND aml.id = aml2.id
         """
     )
+
+    if not openupgrade.column_exists(
+            env.cr, 'account_move_line', 'payment_from_invoice_id'):
```

Comment:
Can you use `openupgrade.get_legacy_name('payment_from_invoice_id')` so that the legacy character of this table is recognizable at database cleanup time?